install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: istio-cleanup-secrets-{{ .Release.Namespace }}
```

Comment:
How about use folder name as `cleanup-secrets` and change the file name to `clean-clusterrole.yaml`

ditto for others